tests/test_datasets/test_internal_datasets.py

```diff
@@ -209,7 +209,7 @@ def test_not_present_part():
     ],
 )
 def test_dataset_statistics(dataset_name, expected_shape, expected_min_date, expected_max_date, dataset_parts):
-    ts_full = load_dataset(dataset_name, parts="full")
+    ts_full = load_dataset(dataset_name, parts="full", rebuild_dataset=True)
```

Comment:
Let's add it into `test_df_exog_statistics` tool

Reply:
It will increase the time of the tests. I think we can use `rebuild_dataset=True` only in `test_dataset_statistics`, because after this test all datasets will be updated.